de_docs/advanced/simnet.md

```diff
@@ -0,0 +1,68 @@
+# **Simnet**
+
+---
+
+## **Overview**
+
+When developing Decred applications or testing potential changes, it is often extremely useful to have a test network where transactions are actually mined into blocks and difficulty levels are low enough to generate blocks as needed.
```

Comment:
Not sure what the part 'transactions are actually mined into blocks' even means.  Probably can just drop that bit.